libraries/app/include/graphene/app/api.hpp

```diff
@@ -185,31 +185,29 @@ namespace graphene { namespace app {
                                                                         unsigned limit = 100,
                                                                         uint32_t start = 0) const;
          /**
-          * @brief Get history of orders with b:a pair
+          * @brief Get list of orders with b:a pair
           * @param a asset in pair
           * @param b asset in pair
-          * @param limit Maximum number of orders to retrieve
+          * @param limit Maximum limit of orders to retrieve
```

Comment:
Same as above.

Reply:
"Maximum count of asset pairs" instead of "Maximum limit of orders", IMHO its better for clear understanding. Am I wrong here or not ? Maybe I've missed something here ?